src/main/kotlin/blackjack/Main.kt

```diff
@@ -1,25 +1,40 @@
 package blackjack
 
 import blackjack.domain.BlackJackGame
-import blackjack.domain.Player
+import blackjack.domain.player.Customer
+import blackjack.domain.player.Dealer
 import blackjack.view.InputView
 import blackjack.view.OutputView
 
 fun main() {
     val inputView = InputView()
     val outputView = OutputView()
 
-    val players = inputView.getPlayerNames().map { Player(it) }
+    val dealer = Dealer()
+    val players = inputView.getPlayerNames().map { Customer(it) }
     val blackJackGame = BlackJackGame()
 
-    blackJackGame.startGame(players)
-    outputView.renderStartGame(players)
+    blackJackGame.startGame(listOf(dealer, players))
```

Comment:
별도 메소드를 만들지 않고 아래와 같이도 사용 가능하겠네요.

```suggestion
    blackJackGame.startGame(players.plus(dealer))
```